docs/docs/APIGuide/Module.md

```diff
@@ -224,6 +226,8 @@ Use `predict` or `predictClass` or `predict_class` on model for Prediction. `pre
 
 Please note that the sequence and the partitions of the output rdd will keep the same with input. So you can zip the output rdd with input rdd to get a (data, result) pair rdd.
 
+For most cnn models, you can set `bigdl.engineType` as **mkldnn** to get better performance when doing prediction.
```

Comment:
For most CNN models, it's recommended to enable MKL-DNN acceleration by specifying `bigdl.engineType ` as 'mkldnn'

Reply:
ok, changed